float64 px
float64 py
float64 pz
float64 roll
float64 pitch
float64 yaw
float64 f1
float64 f2
float64 f3
float64 pre
float64 spt1
float64 spt2
float64 spt3
